tests/functional/s3/test_cp_command.py

```diff
@@ -18,16 +18,16 @@
 from awscli.customizations.s3.utils import relative_path
 from awscli.testutils import BaseAWSCommandParamsTest
 from awscli.testutils import capture_input, mock
-from awscli.compat import six, OrderedDict
+from awscli.compat import BytesIO, OrderedDict
 from tests.functional.s3 import (
     BaseS3TransferCommandTest, BaseS3CLIRunnerTest, BaseCRTTransferClientTest
 )
-
+from tests import requires_crt
```

Comment:
Same here, looks like an unused import

Reply:
Same note, this got auto-added with the cherry-pick. v2 requires the CRT so this isn't needed. In the future once we get `ruff` in place this will be handled automatically for us.